register_users.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+#!/usr/bin/python
+
+import urllib2, random, requests, urllib2, socket, socks, sys
+from bs4 import BeautifulSoup
+from time import sleep
+from lxml import html
+from faker import Faker
+fake = Faker()
+
+url = 'https://github.com/join'
+
+
+def create_connection(address, timeout=None, source_address=None):
+    sock = socks.socksocket()
+    sock.connect(address)
+    return sock
+
+
+def verifyTor():
+    global pure_ip
+    ipcheck_url = 'http://checkip.amazonaws.com/'
+    pure_ip = requests.get(ipcheck_url).text.replace('\n','')
+    socks.setdefaultproxy(socks.PROXY_TYPE_SOCKS5, '127.0.0.1', 9050)
+    socket.socket = socks.socksocket
+    socket.create_connection = create_connection
+
+    tor_ip = urllib2.urlopen(ipcheck_url).read().replace('\n','') # Tor IP
+    if pure_ip == tor_ip:
+        print '[-] Unsuccessful Tor Connection'
+       	sys.exit(1)
+    else:
+        print '[+] Tor Running Normally'
```

Comment:
Please don't use Tor here. We'll separate this ability, if needed into a different module (and certainly a new file). Let's leave this for another PR.